src/metabase/db.clj

```diff
@@ -262,8 +262,8 @@
 
   ([driver :- s/Keyword, details :- su/Map]
    (log/info (u/format-color 'cyan (trs "Verifying {0} Database Connection ..." (name driver))))
+   (classloader/require 'metabase.driver.util)
```

Comment:
Why was this moved out of the `assert`?

Reply:
It doesn't really make a difference actually, if it throws an Exception it will make no difference whether it was in the `assert` or not. I moved it up to make it clearer that the `binding` isn't intended to have an effect on that expression